lc0/build.sh

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash
 
 rm -fr build
-CC=clang CXX=clang++ meson build --buildtype release # -Db_ndebug=true
-# CC=clang CXX=clang++ meson build --buildtype debugoptimized -Db_asneeded=false
-# CC=clang CXX=clang++ meson build --buildtype debug
-cp testdata/* build
-cd build
-ninja
+mkdir build
+pushd build
+cmake .. -DCMAKE_BUILD_TYPE=Release
```

Comment:
It's worth adding comment which other options are possible.
(I need debugoptimized, as that's on what I usually run profiler)
Also is NDEBUG defined by default? It would be nice to have a comment how to control that (it's faster with NDEBUG because there are asserts here and there, but for myself I like to have it)